src/backend/src/services/ws/index.js

```diff
@@ -98,4 +99,15 @@ export default class WebSocketManager {
     });
     return players;
   }
+
+  findGame(ws) {
+    let gameID = '';
+    // Find a game that hasn't started yet
+    this.games.forEach((value, key) => {
```

Comment:
Filter out all the games in round 0 and randomly pick one. Current way would only populate one lobby.

Reply:
Possible concurrency problems, games in filtered list could start and before/while being chosen with random.